### PR TITLE
Readme changes

### PR DESCRIPTION
**Pull request by Dulluhan**
Added some more information and content to the README.

### CODE REVIEW

**Comment by garethellis0** [comment]
Looks like we have some merge conflicts in `README.md` that need to be resolved before we merge.

File: README.md [hunk 1]
```diff
@@ -1,11 +1,173 @@
...
+
+Rostinstall is addressed and run in `install_dependencies.sh`, should there be any need to run manual, rosinstall should be run after all steps from installation and setup has been completed. 
+
+## Conventions
+
+### Github Conventions
+- We follow the Forking Workflow: here is what it is [here](https://www.atlassian.com/git/tutorials/comparing-workflows#forking-workflow) and how to use it [here](https://gist.github.com/Chaser324/ce0505fbed06b947d962)
+- Only commit files that are essential for the system to run; do not put any photos or videos in here
+- All files **must** be formatted properly. Formatting will be enforced with the `clang-format` tool. 
+    - To check and fix formatting, from the `Snowflake` folder run `./clang_format/fix_formatting.sh BRANCH_NAME`, where `BRANCH_NAME` is the name of the branch you intend to merge your code into (ex. `iarrc` or `core`). This script will fix any improperly formatted code, but will refuse to change any files with uncommited changes (to prevent you losing work)
```
Comment (garethellis0) [inline README.md]: we could perhaps change "`iarrc` and `core`" to `master` (just reduce potential confusion as to why neither of the branches given as examples exist in the repo)

Comment (garethellis0) [reply README.md]: `Snowflake` should be changed to `Jormungandr` (I'm just nothing this one instance, but all instances should be changed)

File: README.md [hunk 2]
```diff
@@ -1,11 +1,173 @@
...
+  - [Running Phidget IMU](#running-phidget)
+  - [Running Camera Driver](#cammera-driver)
 
-![alt tag](https://travis-ci.org/ubc-subbots/Jormungandr.svg?branch=master)
 
+## Setup 
+
+### Rosinstall
+
+Rostinstall is addressed and run in `install_dependencies.sh`, should there be any need to run manual, rosinstall should be run after all steps from installation and setup has been completed.
```
Comment (garethellis0) [inline README.md]: `Rostinstall` -> `RosInstall`

File: README.md [hunk 3]
```diff
@@ -1,11 +1,173 @@
...
+ */
+```
+
+- Classes are **CamelCase**
+- Variables are **non_camel_case**
+- Constants are **ALL_CAPS_WITH_UNDERSCORES**
+- Functions are **camelCase**
+- Indentations are 4 spaces
+
+- `CMakeLists.txt` files should be reduced to only contain the minimum amount of comments. The version in `sample_package` has all the comments left in (for the sake of verbosity), so for a more representative example of what yours should look like, see `sb_vision/CMakeLists.txt` (or really any package aside from `sample_package`)
```
Comment (garethellis0) [inline README.md]: `sb_vision` doesn't exist in this repo, but we could replace it with a link to the `sb_vision/CMakeLists.txt` in the Snowflake repo

File: README.md (hunk 2 above)
Comment (Dulluhan) [reply README.md]: Why?

Comment (garethellis0) [reply README.md]: Sorry, the capitalization doesn't really matter, to me, but there's an extra `t` (4th character) in there.

File: README.md (hunk 3 above)
Comment (Dulluhan) [reply README.md]: Done, edited

File: README.md (hunk 2 above)
Comment (Dulluhan) [reply README.md]: Fixed this as well

**Pull request merged**